File: src/items/AutoPart.java
package items;

import java.math.BigDecimal;

public class AutoPart extends Item{
    public AutoPart(BigDecimal price, String name, String description) {
        super(price, name, description);
    }

    public void showSupportedModels(){
        System.out.println("\nHonda Civic V,\nHonda Accord VI");
    }

    public void printWarrantySeal(){
        System.out.println("ACBXMKJLLI142428AGHSD");
    }
}

File: src/items/AutoPart.kt
package items

import java.math.BigDecimal

open class AutoPart(price: BigDecimal, name: String, description: String) : Item(price, name, description) {
    fun showSupportedModels() {
        println("\nHonda Civic V,\nHonda Accord VI")
    }

    fun printWarrantySeal() {
        println("ACBXMKJLLI142428AGHSD")
    }
}